Find a functionally equivalent code:android.view.ViewRootImpl.outputDisplayList:COMMENT
Method Modifier: hidden      
Comment:/**
 * @hide
 */

@@ -1,8 +1,6 @@
 {
-    if (mAttachInfo != null && mAttachInfo.mHardwareCanvas != null) {
-        DisplayList displayList = view.getDisplayList();
-        if (displayList != null) {
-            mAttachInfo.mHardwareCanvas.outputDisplayList(displayList);
-        }
+    RenderNode renderNode = view.getDisplayList();
+    if (renderNode != null) {
+        renderNode.output();
     }
 }

